frontend/hub/api/utils.tsx

```diff
@@ -8,6 +8,11 @@ import {
   putHubRequest,
 } from './request';
 
+export function getBaseAPIPath() {
```

Comment:
Why do we need getBaseAPIPath when there is hubAPI?

Reply:
Updated.